src/optimalcontrol/src/ConstraintsGroup.cpp

```diff
@@ -177,16 +223,26 @@ namespace optimalcontrol {
             return true;
         }
 
+        std::vector<TimeRange>& ConstraintsGroup::getTimeRanges()
+        {
+            if (m_pimpl->timeRanges.size() != numberOfConstraints())
```

Comment:
@traversaro  will complain that it is always better to add the scope brackets (`{}`) even for a one-line statement

Reply:
I think I did it many times. For your amusement :grin: